NEWS.md

```diff
@@ -1,3 +1,37 @@
+### 7.0.0 (2020-11-09):
+
+* Drop Node v8.x support and add Node v14.x
+  * Removed Node v8.x and added v14.x to CI
+  * Adds check that minimum Node version is >= 10
+  * Sets Node engine to >= 10 and < 15
```

Comment:
I hope we aren't doing this? I asked that we not set the engine version to < 15 because it prevents folks from trying the agent early on future versions such as when 16 is released.

Reply:
It doesn't prevent them, just [warns them](https://github.com/newrelic/node-newrelic/pull/524/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R60)

Reply:
OK good. This release message is wrong then. Would remove the `<15` from this line.

Reply:
fixed